data_transfer/get_project_metadata.py

```diff
@@ -0,0 +1,114 @@
+import csv
+import json
+import logging
+import sys
+
+import click
+from sample_metadata.apis import FamilyApi, ParticipantApi, SampleApi
+
+PAPI = ParticipantApi()
+SAPI = SampleApi()
+FAPI = FamilyApi()
+
+
+def get_individual_metadata(project: str):
+    """Returns all rows of seqr individual metadata for a project"""
+    return PAPI.get_individual_metadata_for_seqr(project).get('rows')
+
+
+def get_hpo_terms(individual_metadata: list[dict]):
+    """Returns a mapping of individual ID to list of hpo terms"""
+    indiv_hpo_terms = {}
+    for row in individual_metadata:
+        indiv_id = row.get('individual_id')
+        hpo_terms = ''
+        if row.get('hpo_terms_present'):
+            hpo_terms = row.get('hpo_terms_present').split(',')
+
+        indiv_hpo_terms[indiv_id] = hpo_terms
+
+    return indiv_hpo_terms
+
+
+def get_pedigrees(project: str):
+    """Get all pedigree rows for a project"""
+    return FAPI.get_pedigree(project)
+
+
+def get_sample_map(project: str):
+    """Returns the internal id : external id sample map for a project"""
+    return SAPI.get_all_sample_id_map_by_internal(project)
+
+
+def write_outputs(
+    individual_hpo_terms: dict[str, list],
+    pedigrees: list[dict],
+    sample_map: dict[str, str],
+    output_path: str,
+):
+    """Writes the HPO terms file, pedigree file, and sample map file to the output path"""
+
+    # HPO terms tsv
+    with open(f'{output_path}/hpo_terms.tsv', 'w') as f:
```

Comment:
It's good practice to open files with a specific encoding, a more stringent linter would complain

```suggestion
    with open(f'{output_path}/hpo_terms.tsv', 'w', encoding='utf-8') as f:
```